build/docker/keycloak/keycloak.crt

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Why we need to regenerate the certificated? I don't see any problem in that, but I'm just curious about it.

Reply:
@jespino The old certificate wasn't correct...didn't work. If you look at the certificates in the realm.json file. The old one doesn't match any of them. The "new" certificate, is just the certificate from the realm.json file. It will now work "out of the box".
https://github.com/mattermost/mattermost-server/blob/master/build/docker/keycloak/realm.json#L1431

Reply:
Perfect! Thanks for the explanation.